### DISCUSSION TITLE
[firebase_ui_firestore][flutter] How to search new query and refresh to FirestoreListView

### DISCUSSION THREAD

**Discussion by matekhong** [firebase/flutterfire#11223] [Q&A]
I've created FirestoreListView as below

![image](https://github.com/firebase/flutterfire/assets/17153200/a989f6c6-6b81-4e02-b39a-ea4aeb3a0c12)

  
  But now I want to create SearchField to filter the result in FirestoreListView how could I refresh query variable?

**Comment by Al-Krause-ex** [comment]
@matekhong 
Did you find answer?
